lua/entities/gmod_tardis/modules/sh_falling.lua

```diff
@@ -0,0 +1,148 @@
+            end
+
+            local angv_fast = (angv.x > 50 or angv.y > 50)
+
+            if (math.abs(ang.p) > 5 or math.abs(ang.r) > 5) and not angv_fast then
+                ph:ApplyForceOffset( up * -ang.p * 5, cen - fwd2 * lev)
+                ph:ApplyForceOffset(-up * -ang.p * 5, cen + fwd2 * lev)
+                ph:ApplyForceOffset( up * -ang.r * 5, cen - ri2 * lev)
+                ph:ApplyForceOffset(-up * -ang.r * 5, cen + ri2 * lev)
+            else
+                if angv_fast then
+                    ph:SetAngleVelocityInstantaneous(Vector(angv.x * 0.9, angv.y * 0.9, angv.z))
+                end
+            end
+        elseif falling and vel.z >= 0 then
+
+            if not stop_time then
+                self:SetData("falling_stop_time", CurTime())
+                self:SendMessage("fall_landing", { self:GetData("falling_hard") } )
+                self:SetData("falling_hard", nil)
+            elseif CurTime() - stop_time > 1 and not self:GetData("mat") then
+                self:SetData("falling", false, true)
+            end
+
+            if math.abs(ang.p) <= 45 and math.abs(ang.r) <= 45 and vel.z > 0 then
+                local newvel_x = math.Clamp(vel.x * 0.05,-30,30)
+                local newvel_y = math.Clamp(vel.y * 0.05,-30,30)
+
+                ph:SetVelocityInstantaneous(-up * 50 * phm)
+                ph:AddVelocity(Vector(newvel_x,newvel_y,0))
+
+                local newavel_x = math.Clamp(angv.x * 0.1,-300,300)
+                local newavel_y = math.Clamp(angv.y * 0.1,-300,300)
+                local newavel_z = math.Clamp(angv.z * 0.1,-300,300)
+                ph:SetAngleVelocityInstantaneous(Vector(newavel_x,newavel_y,newavel_z))
+            end
+        end
+    end)
+else
+    ENT:OnMessage("fall_landing", function(self, data, ply)
+        if not TARDIS:GetSetting("sound") then return end
+
+        local hard = data[1]
+        local snds = self.metadata.Exterior.Sounds
+
+        if TARDIS:GetSetting("flight-externalsound") then
+            if not hard then
+                self:EmitSound(snds.FlightLand)
+            else
+                self:EmitSound(snds.FlightFall)
+            end
+        end
+
+        if IsValid(self.interior) and TARDIS:GetSetting("flight-internalsound") then
+            local snds_i = self.metadata.Interior.Sounds
+            self.interior:EmitSound(snds_i.FlightLand or snds.FlightLand)
+        end
+    end)
+
+    ENT:AddHook("Think", "falling", function(self)
+        if self:GetData("free_movement")
+            and TARDIS:GetSetting("flight-externalsound")
+            and TARDIS:GetSetting("sound")
+        then
+            local p=math.Clamp(self:GetVelocity():Length()/125,0,15)
+            if self.fallsound and self.fallsound:IsPlaying() then
+                local ply=LocalPlayer()
+                local e=ply:GetViewEntity()
+                if not IsValid(e) then e=ply end
+                if e:EntIndex()==-1 then -- clientside prop
```

Comment:
Why do we need the clientside prop checks here or anything outside of `ply:GetTardisData("outside" / "exterior")?

I noticed that even in third person it still uses the doppler version of the effect and to my understanding it should just be the normal one in that scenario, similar to the flight sound in third person

Reply:
To be honest I'm not very sure about how the doppler sound is to work. 
My intention was to make the wind sound change its pitch as the speed increases - and since the flight sound works realistically, I just kept everything. Should this part not be there?